sphinx/conf.py

```diff
@@ -0,0 +1,68 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+import os
+import sys
+
+import sphinx.builders.html
+import sphinx.builders.latex
+import sphinx.builders.texinfo
+import sphinx.builders.text
+import sphinx.ext.autodoc
+
+# Append project source directory to path environment variable
+sys.path.insert(0, os.path.abspath("../src"))
+
+django_settings = "voctrainer.settings"
+
+
+# -- Project information -----------------------------------------------------
+
+project = 'Lunes CMS'
+copyright = '2021, Lunes'
+author = 'Lunes'
+
+# The full version, including alpha/beta/rc tags
+release = '1.0'
+
+
+# -- General configuration ---------------------------------------------------
+
+# Add any Sphinx extension module names here, as strings. They can be
+# extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
+# ones.
+extensions = [
+    "sphinx.ext.autodoc",
```

Comment:
```suggestion
```
I think this can be omitted, since it is already set up in [sphinxcontrib_django2](https://github.com/timoludwig/sphinxcontrib-django2/blob/c5faedc3a1fc54ce8b559b37f150b9d52d19d5c5/sphinxcontrib_django2/docstrings/__init__.py#L68).